### PR TITLE
Resource models with a foreign key

### PR DESCRIPTION
**Pull request by AndrewTelleria**
Fixes issue #115 

Changes: Add two models to the models file in the home app. The first model is a Resource class that has a language field and the second model is a Resources class that pools the Resource class.

### CODE REVIEW

File: home/models.py [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+	language = models.CharField(max_length=30)
+	example = models.TextField()
+
+	def __str__(self):
+		return "%s" % (self.language)
+
+class Resources(models.Model):
+	resource = models.ForeignKey(Resource, on_delete=models.CASCADE)
+	headline = models.CharField(max_length=100)
+	pub_date = models.DateField()
```
Comment (divyam3897) [inline home/models.py]: pub date not required.

File: home/models.py [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+	image = models.ImageField() 	# Might need to install Pillow for ImageField to work.
+	language = models.CharField(max_length=30)
+	example = models.TextField()
+
+	def __str__(self):
+		return "%s" % (self.language)
+
+class Resources(models.Model):
+	resource = models.ForeignKey(Resource, on_delete=models.CASCADE)
+	headline = models.CharField(max_length=100)
```
Comment (divyam3897) [inline home/models.py]: give it a better name.

File: home/models.py [hunk 3]
```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
-# Create your models here.
+
+class Resource(models.Model):
+	""" A resource that shows students the name of the language, example, and an image. """
+	image = models.ImageField() 	# Might need to install Pillow for ImageField to work.
+	language = models.CharField(max_length=30)
+	example = models.TextField()
```
Comment (divyam3897) [inline home/models.py]: what is example?

**Comment by divyam3897** [comment]
@AndrewTelleria still on it?

File: home/models.py (hunk 3 above)
Comment (AndrewTelleria) [reply home/models.py]: The example is example code for the specified language.

File: home/models.py (hunk 2 above)
Comment (AndrewTelleria) [reply home/models.py]: ResourcesPool? Is that a better name?

File: home/models.py (hunk 1 above)
Comment (AndrewTelleria) [reply home/models.py]: Ok.

**Comment by AndrewTelleria** [comment]
I am still on it. I had to work this weekend and didn't have time.

**Comment by AndrewTelleria** [comment]
I have changed the name of the Resources model to ResourcesPool and I removed the pub_date and example. I do not know what else you want to represent with these models.

File: README.md [hunk 4]
```diff
@@ -97,6 +98,7 @@ After selecting an issue
  5. The pull request will not get merged until and unless the commits are squashed. In case there are multiple commits on the PR, the commit author needs to squash them and not the maintainers cherrypicking and merging squashes.
  6. If the PR is related to any front end change, please attach relevant screenshots in the pull request description
 o
+
```
Comment (divyam3897) [inline README.md]: why these changes?

**Comment by AndrewTelleria** [comment]
The PEP8 guidlines are followed, the class names are supposed to have CapWords which they are. If you see anything that is not followed by the PEP8 guidelines please point it out.

**Comment by AndrewTelleria** [comment]
As far as squashing the commits I am having trouble. I went to pull from the master branch and it made me make a new commit. When I go to rebase my commit history the last two commits will not show up in my log.

**Comment by AndrewTelleria** [comment]
There was a merge conflict that I had to resolve in the README before I could push. I copied and pasted the README from the current master branch into my branch's README hoping that wouldn't have caused any conflict. My branch's README should be identical to the master's README.

File: home/models.py [hunk 5]
```diff
@@ -1,3 +1,20 @@
...
+	image = models.ImageField() 
+	language = models.CharField(max_length=30)
+
+	def __str__(self):
+		return "%s" % (self.language)
+
+		
+class ResourcesPool(models.Model):
+	resource = models.ForeignKey(Resource, on_delete=models.CASCADE)
+	headline = models.CharField(max_length=100)
```
Comment (divyam3897) [inline home/models.py]: can you think of a better name then headline?
